### DISCUSSION TITLE
v2.1.7

### DISCUSSION THREAD

**Discussion by the1812** [the1812/Bilibili-Evolved#3156] [分享 / Share]
✨**正式版** `v2.1.7`
- 修复 `简化直播间` 里带头像框的头像大小还是不对. (#3119)
- 修复 `清爽首页` 无法设置最大宽度. (#3120)
- 检查更新时的 `安装` 链接现在将遵循更新源设置. (#3122)
- 修复 `BV 号转换` 在番剧区产生的链接错误. (#3123)
- 修复 `查看封面` 在番剧区显示错误. (#1992)
- 修复 `自定义顶栏` 的透明模式在 b 站春季主题下失效. (#3126)
- 修复更改了主题色后文字颜色没有更新.
- 修复 `下载视频` 中的下拉菜单被遮挡. (#3129, #3133)
- 更新合集的一些夜间模式样式. (#3027)
- `传统连播模式` 支持在合集的最后一集停止连播. (#2940)
- 修复番剧区在文件命名时取得的 `title` 变量错误. (#3140)
- 修复通过 `快捷键扩展` 进行的静音操作提示总是显示 `已静音`. (#2830)
- 修复 `自定义顶栏` 的历史弹窗不支持触摸模式. (#3142)
- `下载视频` 的编码格式限制恢复之前的逻辑, 仍然是优先匹配, 没有相应编码的视频源时自动回退到其他编码. (#3118)
- 完善 `mountVueComponent` 的类型. (PR #3151 by [timongh](https://github.com/timongh))
- 修复 Wasm 播放器部分情况下识别错误. (#3131)

----

✨**预览版** `v2.1.7-preview`
- `倍速增强` 正式完成分离, 变更为 `记忆倍速`, `扩展倍速` 和独立的插件 `视频倍速 - 快捷键支持`, 详见 [PR 说明](https://github.com/the1812/Bilibili-Evolved/pull/2746). (PR #2746 by [JLoeve](https://github.com/LonelySteve))
- 为通用设置增加了一些选项描述. (#3140)
- 新增 ScopedConsole API, 可以创建带有固定前缀的 `console` 对象. 同时脚本作用域中的 `window.console` 将自带 `Bilibili Evolved` 前缀. (#3105)
```ts
import { useScopedConsole, randomScopedConsole } from '@/core/utils/log'

console.log(123)
// [Bilibili Evolved] 123
{
  const console2 = useScopedConsole('scope')
  console2.log(123)
  // [Bilibili Evolved] [scope] 123

  const console3 = randomScopedConsole({ console: console2 })
  console3.log(123)
  // [Bilibili Evolved] [scope] [977a7962] 123
}
```

<hr /><em>This discussion was created from the release <a href='https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.1.7'>v2.1.7</a>.</em>

**Comment by the1812** [comment]
@FoundTheWOUT 你之前说的使用这个 https://github.com/peter-evans/repository-dispatch 来通知文档更新, `event-type` 应该写什么? 以及是否需要在 `client-payload` 里带上什么参数?

**Reply by FoundTheWOUT** [reply]
我改了下项目，现在支持从远端获取文件然后重新构建了。不需要这个 dispatch 了，只要在发版的时候触发下构建 hook 就行。

**Comment by DDuann027** [comment]
为什么下载后还是2.16   功能面变也是 空空如也哦 =￣ω￣=

**Reply by Geeyun-JY3** [reply]
你下的不会又是本 Release 的 zip 文件吧

**Reply by DDuann027** [reply]
> 是的  是我下载错了吗？

**Reply by JulyCalm** [reply]
我的也是空空如也，我是从主页给的正式版链接安装的。。。

**Reply by Geeyun-JY3** [reply]
为什么不先认真看看 README 呢：https://github.com/the1812/Bilibili-Evolved#设置

#1301
> # 下载 zip 后安装的版本不对
> 本项目从来没有从 zip 安装的说法, 只有[首页](https://github.com/the1812/Bilibili-Evolved#%E5%AE%89%E8%A3%85)中的安装链接才是正确的.

为什么今年以来出现了这么多从 Release 下载的人（

**Reply by Geeyun-JY3** [reply]
@JulyCalm 你也需要认真地看看 README 呢：[https://github.com/the1812/Bilibili-Evolved#设置](https://github.com/the1812/Bilibili-Evolved#%E8%AE%BE%E7%BD%AE)

**Reply by DDuann027** [reply]
> @JulyCalm 你也需要认真地看看 README 呢：[https://github.com/the1812/Bilibili-Evolved#设置](https://github.com/the1812/Bilibili-Evolved#%E8%AE%BE%E7%BD%AE)

谢谢   好好看

**Reply by DDuann027** [reply]
> 为什么不先认真看看 README 呢：[https://github.com/the1812/Bilibili-Evolved#设置](https://github.com/the1812/Bilibili-Evolved#%E8%AE%BE%E7%BD%AE)
> 
> #1301
> 
> > # 下载 zip 后安装的版本不对
> > 本项目从来没有从 zip 安装的说法, 只有[首页](https://github.com/the1812/Bilibili-Evolved#%E5%AE%89%E8%A3%85)中的安装链接才是正确的.
> 
> 为什么今年以来出现了这么多从 Release 下载的人（

不能爬出去了  404

**Reply by JulyCalm** [reply]
@Geeyun-JY3 弄好了，感谢

**Reply by DDuann027** [reply]
> @Geeyun-JY3 弄好了，感谢

怎么弄的啊  在线求教

**Reply by Geeyun-JY3** [reply]
> 不能爬出去了 404

网络问题请自行解决。

**Reply by DDuann027** [reply]
> > 不能爬出去了 404
> 
> 网络问题请自行解决。

谷歌访问助手 又被禁了

**Reply by JulyCalm** [reply]
@DDuann027 你得在设置里的组件，插件，样式里的在线按钮里下载你需要的功能包，之后所有的你的自定义功能都在设置里调整

**Reply by Geeyun-JY3** [reply]
@DDuann027 安装方法 [README](https://github.com/the1812/Bilibili-Evolved#安装) 不是写得很清楚吗？爬不出去，jsDelivr 用不了吗？

**Reply by DDuann027** [reply]
> @DDuann027 安装方法 [README](https://github.com/the1812/Bilibili-Evolved#%E5%AE%89%E8%A3%85) 不是写得很清楚吗？爬不出去，jsDelivr 用不了吗？

fotiaoqiang 出去了

**Reply by DDuann027** [reply]
> @DDuann027 你得在设置里的组件，插件，样式里的在线按钮里下载你需要的功能包，之后所有的你的自定义功能都在设置里调整

我明白这操作   就是下载了功能包 插件 组件    但还是空空如也  ，不知道是不是我的油猴版本低了 4.12

**Reply by Geeyun-JY3** [reply]
建议检查浏览器兼容性，不支持内核太旧的浏览器：https://github.com/the1812/Bilibili-Evolved#兼容性

**Reply by DDuann027** [reply]
> 建议检查浏览器兼容性，不支持内核太旧的浏览器不支持：[https://github.com/the1812/Bilibili-Evolved#兼容性](https://github.com/the1812/Bilibili-Evolved#%E5%85%BC%E5%AE%B9%E6%80%A7)

刚刚更新Chrome 已是最新版本
版本 99.0.4844.82（正式版本）

**Reply by FoundTheWOUT** [reply]
> 不能爬出去了 404

网络问题可以访问 [bilibili-evolved-doc](https://bilibili-evolved-doc.vercel.app/user/install) 查看文档

**Reply by Geeyun-JY3** [reply]
那要不先升升 Tampermonkey？正式版最新都到 4.15 了。

**Reply by DDuann027** [reply]
> 那要不先升升 Tampermonkey？正式版最新都到 4.15 了。

升级了 安装了组件 插件 样式   浏览器重启、刷新  还是空空如也

**Reply by DDuann027** [reply]
> 那要不先升升 Tampermonkey？正式版最新都到 4.15 了。

能帮我看看吗？q ：76996419

**Reply by Geeyun-JY3** [reply]
帮不了你，如果排查过其他脚本或扩展都还是不行，建议按着指引开 issue。

**Reply by DDuann027** [reply]
> 帮不了你，如果排查过其他脚本或扩展都还是不行，建议按着指引开 issue。

兄弟！ Edge高好了，一样的操作虽然不明白Chrome不行，看来是时候告别Chrome了

**Comment by Geeyun-JY3** [comment]
设置半天搞蒙了，文本颜色设为 `自动` 时，文本颜色和主题色有关系吗？和夜间模式有关系吗？

**Reply by the1812** [reply]
有, 主题色的亮度较低时, 会变成黑色, 夜间模式许多主题色的按钮都适配了这个机制. 这是一个 v1 的遗留功能, 只是发现了之前无论主题色怎么设置都是白色的 bug.

<img src="https://user-images.githubusercontent.com/26504152/159628059-80da04d3-a889-4610-8b62-96c5d6ab3434.png" width="200">
<img src="https://user-images.githubusercontent.com/26504152/159628087-d81f6937-88de-40d9-ad6b-325e190a656f.png" width="200">

**Reply by Geeyun-JY3** [reply]
夜间模式下文本也变黑感觉怪怪的……

而且 b 站自己的天蓝色按钮的文本也是白的，没开夜间模式下会不会显得不太协调……

**Comment by LZHPW** [comment]
不知道为什么，我下载后总是出问题，上次是播放器页面 黑暗模式 出问题，这次是动态页面

**Reply by Geeyun-JY3** [reply]
善用搜索，新版播放页和动态页面都没适配的：#2615、#3191

**Comment by quabc** [comment]
夜间模式动态没变黑

**Reply by Geeyun-JY3** [reply]
请
善
用
搜
索
，
看
置
顶
#3191

**Comment by RuJingTingShan** [comment]
记忆倍速 - 倍速还原操作失败
每打开一个b站页面就弹一遍，怎么关掉啊，已经卸载了插件，再安装还是弹

**Reply by LonelySteve** [reply]
确定卸载了「记忆倍速」插件？有截图吗？

**Comment by Malus-Mill** [comment]
表示倍数播放1拆3之后、变卡了

**Comment by St-Jiuyu** [comment]
可以加一个直播间仅播放声音吗 拜托啦

**Reply by Geeyun-JY3** [reply]
请善用搜索：#2394

**Reply by St-Jiuyu** [reply]
主要吧 就是用的热点 网络状况很差 切换选项卡和最小化也不能正常听直播…… 模拟器仅播声音倒是勉强能用 热点带宽有限
